Client/Protocols/CanRemoveQuickActionBookmark.swift

```diff
@@ -15,14 +15,24 @@ extension CanRemoveQuickActionBookmark {
     func removeBookmarkShortcut() {
         // Get most recent bookmark
         profile.places.getRecentBookmarks(limit: 1).uponQueue(.main) { result in
-            guard let bookmarkItems = result.successValue else { return }
+            guard let bookmarkItems = result.successValue else {
```

Comment:
Can you add tests for this?

Reply:
Done! Here's what I did in that last commit:
- Put quick actions infos in a struct (i.e. keys and versions)
- Quick actions is no longer a shared instance (singleton no more)
- Add a protocol for quick actions, to unit tests easily
- Indent properly to make lines shorter